networkx/generators/classic.py

```diff
@@ -489,29 +489,40 @@ def cycle_graph(n, create_using=None):
 
 @nx._dispatchable(graphs=None, returns_graph=True)
 def dorogovtsev_goltsev_mendes_graph(n, create_using=None):
-    """Returns the hierarchically constructed Dorogovtsev-Goltsev-Mendes graph.
+    r"""Returns the hierarchically constructed Dorogovtsev--Goltsev--Mendes graph.
 
-    The Dorogovtsev-Goltsev-Mendes [1]_ procedure produces a scale-free graph
-    deterministically with the following properties for a given `n`:
-    - Total number of nodes = ``3 * (3**n + 1) / 2``
-    - Total number of edges = ``3 ** (n + 1)``
+    The Dorogovtsev--Goltsev--Mendes [1]_ procedure deterministically produces a
+    scale-free graph with $\frac{3}{2} (3^{n-1} + 1)$ vertices
+    and $3^n$ edges for a given $n$.
+
+    Note that $n$ denotes the number of times the state transition is applied,
+    starting from the base graph with $n = 0$ (no transitions), as in [2]_.
+    This is different from the parameter $t = n - 1$ in [1]_.
 
     .. plot::
 
-        >>> nx.draw(nx.dorogovtsev_goltsev_mendes_graph(3))
+        >>> nx.draw_planar(nx.dorogovtsev_goltsev_mendes_graph(3))
 
     Parameters
     ----------
     n : integer
-       The generation number.
+       The generation number. Must be greater than or equal to 0.
 
-    create_using : NetworkX Graph, optional
-       Graph type to be returned. Directed graphs and multi graphs are not
+    create_using : NetworkX graph constructor, optional (default `nx.Graph`)
+       Graph type to be returned. Directed graphs and multigraphs are not
        supported.
+       If this is a graph instance, it gets cleared, then populated.
```

Comment:
Can we match the text for the previous couple of functions for create_using?
The bit about directed and multigraph is covered below now that you added a "Raises" section.

```
    create_using : NetworkX graph constructor, optional (default=`nx.Graph`)
       Graph type to create. If graph instance, then cleared before populated.
```
```

Reply:
Good point about the "Raises" not requiring duplication anymore. I'll change this.